<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramForm</class>
 <widget class="QWidget" name="ProgramForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>164</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pp_job_description_label">
       <property name="text">
        <string notr="true">Job description</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pp_job_description_lineedit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pp_acidizing_technique_label">
       <property name="text">
        <string notr="true">Acidizing technique</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pp_acidizing_technique_lineedit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pp_design_steps_label">
       <property name="text">
        <string notr="true">Design Steps</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="pp_design_steps_textedit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pp_use_diversion_label">
       <property name="text">
        <string notr="true">Do you use diversion?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="pp_use_diversion_checkbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pp_diversion_type_label">
       <property name="text">
        <string notr="true">Type of diversion</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pp_diversion_type_lineedit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pp_placement_technique_label">
       <property name="text">
        <string notr="true">Placement technique</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pp_placement_technique_lineedit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pp_coil_tube_od_label">
       <property name="text">
        <string notr="true">Coil tube outside diameter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pp_coil_tube_length_label">
       <property name="text">
        <string notr="true">Coil tube length</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pp_coil_tube_od_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="pp_coil_tube_od_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="pp_coil_tube_length_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="pp_coil_tube_length_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
